Docs/source/Software/Tasks.rst

```diff
@@ -2,6 +2,36 @@
 Real Time Operating System Tasks
 ********************************
 
+Initialization Task: Alex Koo
+===============================
+
+Purpose
+    Create tasks by calling "OSTaskCreate()" and provide argument specifying to RTOS how the task will be managed.
+    The order of the semaphore Initialization matters because if the fault semaphore isn't initialized first and a fault is called we do not go into the fault state.  
+
+
+Functionality
+    1) The TCB, Task name, Task function argument, Priority, Stack, Watermark limit for debugging, Stack size, 
+    Queue size, Time quanta, Extension pointer, Options, Return err code is provided. 
+
+    2) Defines the priority level for the tasks
+    
+    3) Specifies the size of the task's stack in number of elements which corresponds to the number of bytes available
+    on the stack 
+
+Priority
+    The initialization has the highest priority. 
+
+Timing Requirements
+    Initialization Task also creates the semaphores which is a programming construct that helps us achieve concurrency 
```

Comment:
The fact that it creates semaphores should be in the description and functionality. You do not have to redefine semaphores because it is already defined in the glossary